src/activities/mirrorgame/ActivityInfo.qml

```diff
@@ -0,0 +1,35 @@
+/* GCompris - ActivityInfo.qml
+ *
+ * Copyright (C) 2016 Your Name <yy@zz.org>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import GCompris 1.0
+
+ActivityInfo {
+  name: "mirrorgame/Mirrorgame.qml"
+  difficulty: 3
+  icon: "mirrorgame/mirrorgame.svg"
+  author: "Shubham Nagaria &lt;shubhanrnagaria@gmail.com;"

```

Comment:
it should end with &gt; (not just ";"). Else, if you go on the help page, you don't have the closing >.